Lib/email/utils.py

```diff
@@ -215,6 +215,12 @@ def parsedate_to_datetime(data):
 
 
 def parseaddr(addr):
+    """
+    Parse address – which should be the value of some address-containing field such as To or Cc
+    – into its constituent realname and email address parts.
+    Returns a tuple of that information, unless the parse fails,
+    in which case a 2-tuple of ('', '') is returned.
+    """
```

Comment:
The style we try to follow for docstrings is a single initial sentence that tries to summarize the most important parts of the API, followed by a blank line, followed by a more extensive description of anything not covered in the initial sentence.  The entire thing should be in imperative mode (we are *not* consistent about that in the docstrings and docs, but that is my understanding of our desired style).

Reply:
My bad. I just copy-pasted the entire thing from the python docs. I'll try to make it more readable

Reply:
Your change doesn't seem to have done anything other than make the line lengths longer than our 79 char standard.  There is a reason why we don't use autodoc: docstrings and the main docs usually have somewhat different wording (and the docstrings are usually shorter, although in this case they may not be).

Reply:
I re-read your instructions carefully and I think I finally got it right. I'll make sure I read the python developer's guide before I further contribute to this language.

Reply:
I'm not sure how much of this docstring detail is in the devguide.  It might be mentioned in PEP 8, or maybe there's a separate informational PEP about docstrings, I forget.